.devcontainer/Wallaroo-Platform-DS/devcontainer.json

```diff
@@ -23,25 +24,13 @@
 			]
 		}
 	},
-	"forwardPorts": [ 8080, 8443, 8081],
+	"forwardPorts": [8443],
```

Comment:
this isn't 8081 and 8443 used by edge right?

Reply:
Correct it's mapped from API lb 443